### PR TITLE
Add try-catch block to POAP API call

### PR DESCRIPTION
**Pull request by garyghayrat**
This simple solution seems to work even when the API url is wrong.

### CODE REVIEW

File: frontend/src/utils/address.ts [hunk 1]
```diff
@@ -147,6 +147,12 @@ const jsonFetch = (url: string) => fetch(url).then((res) => res.json());
...
 // Returns true if the address owns any POAP tokens
 const hasPOAPs = async (address: string) => {
-  const poaps = await jsonFetch(`https://api.poap.xyz/actions/scan/${address}`);
-  return poaps.length > 0 ? true : false;
+  try {
+    const poaps = await jsonFetch(`https://api.poap.xyz/actions/scan/${address}`);
+    return poaps.length > 0 ? true : false;
+  } catch (err) {
+    console.warn('Error in hasPOAPs');
+    console.warn(err);
```
Comment (mds1) [inline frontend/src/utils/address.ts:155]: Reminder to always use `window.logger` in the frontend

File: frontend/src/utils/address.ts [hunk 2]
```diff
@@ -147,6 +147,12 @@ const jsonFetch = (url: string) => fetch(url).then((res) => res.json());
...
+    return false;
```
Comment (mds1) [inline frontend/src/utils/address.ts:156]: Hmm are we ok with assuming `false` when this query fails? Maybe we should default to `true` to be more conservative?

**Review by apbendi — APPROVED** [review]
Looks and works well! Nice @garyghayrat

File: frontend/src/utils/address.ts (hunk 2 above)
Comment (apbendi) [reply frontend/src/utils/address.ts:156]: I don't think so. A false positive could cause a lot of confusion for the user, especially if they're using an address that is totally safe.

Comment (mds1) [reply frontend/src/utils/address.ts:156]: Yea, the third options is return something other tan true/false and say "may have POAPs, not sure, the check failed" and let the user handle that information, but that feels overly complex

Comment (mds1) [reply frontend/src/utils/address.ts:156]: We're doing this "to be helpful" to users—this list is not comprehensive and there's other things that could break privacy, so I think false is ok

File: frontend/src/utils/address.ts (hunk 1 above)
Comment (garyghayrat) [reply frontend/src/utils/address.ts]: Noted! Addressed it here https://github.com/ScopeLift/umbra-protocol/pull/370/commits/ff0b19c2d6e044393dee68cdf5716548d8d5f884

**Review by mds1 — APPROVED** [review]

**Pull request merged**